sdk/src/Services/S3/Custom/Transfer/_async/ITransferUtility.async.cs

```diff
@@ -237,6 +237,37 @@ public partial interface ITransferUtility : IDisposable
         /// <returns>The task object representing the asynchronous operation.</returns>
         Task<Stream> OpenStreamAsync(TransferUtilityOpenStreamRequest request, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
+        /// 	Returns a stream from which the caller can read the content from the specified
+        /// 	Amazon S3 bucket and key, along with response metadata.
+        /// 	The caller of this method is responsible for closing the stream.
+        /// </summary>
+        /// <param name="bucketName">
+        /// 	The name of the bucket.
+        /// </param>
+        /// <param name="key">
+        /// 	The object key.
+        /// </param>
+        /// <param name="cancellationToken">
+        ///     A cancellation token that can be used by other objects or threads to receive notice of cancellation.
+        /// </param>
+        /// <returns>The task object representing the asynchronous operation with response metadata.</returns>
+        Task<TransferUtilityOpenStreamResponse> OpenStreamWithResponseAsync(string bucketName, string key, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        /// 	Returns a stream to read the contents from Amazon S3 as 
```

Comment:
Since this is the main API public documentation this should be expanded with details about the buffering and where to adjust things. With this description I wouldn't see this method any different then existing other then I get a response.